components/Toggle/Toggle.js

```diff
@@ -3,22 +3,16 @@ import React, {Component, PropTypes} from 'react';
 import s from './Toggle.css';
 
 export default class Toggle extends Component {
-  constructor(props) {
-    super(props);
-    this.handleChange = this.handleChange.bind(this);
-  }
-  handleChange() {
-    this.props.onChange(!this.props.value);
-  }
   render() {
-    const isOn = this.props.value === true;
+    const {value, onChange, className} = this.props;
+    const isOn = value === true;
```

Comment:
Why do `isOn` at all if it's only used once. Move this logic onto the label's className.